applications/openshift/authentication/ocp_idp_no_htpasswd/tests/no_file.fail.sh

```diff
@@ -3,3 +3,4 @@
 # remediation = none
 
 rm -rf /kubernetes-api-resources/apis/config.openshift.io/v1/oauths
+echo true
```

Comment:
This looks suspicious

Reply:
This is part of a test commit to trigger a change in the rule so it can be tested by the CI. It will not be part of the final state of this PR. :)